### ISSUE TITLE
Find and identify the code that required to compile the source

### ISSUE THREAD

**Issue by LordGolias** [LordGolias/Brytenwalda#1]
We need to know what files from the module are required by our current source. E.g. maybe some files are there and they don't have to be there.

**Comment by gdwitt** [comment]
It would be easiest to just upload a clean copy without the working materials.
Troops.txt is from Brytenwalda Repolished and I was using it to slowly reverse engineer armors and weapons.

**Comment by gdwitt** [comment]
If you mean the Resource files in the playable module files, there is nothing needed. The correct names need to appear in the source files, but the Resource files are completely separate.
These 4 files are erased and created at every compilation. Variables is a tricky text file. If it's there, it holds onto old variables and triggers that you might have changed. Here is process_init:
from module_info import *
from process_operations import *
import os

print "Initializing..."

try:
  os.remove(export_dir + 'tag_uses.txt')
except:
  a = []
try:
  os.remove(export_dir + 'quick_strings.txt')
except:
  a = []
try:
  os.remove(export_dir + 'variables.txt')
except:
  a = []
try:
  os.remove(export_dir + 'variable_uses.txt')
except:
  a = []

variables = []
variable_uses = []
try:
  file = open("variables.txt","r")
  var_list = file.readlines()
  file.close()
  for v in var_list:
    vv = string.strip(v)
    if vv:
      variables.append(vv)
      variable_uses.append(int(1))
  save_variables(export_dir, variables, variable_uses)
except:
  print "variables.txt not found. Creating new variables.txt file"

**Comment by LordGolias** [comment]
> If you mean the Resource files in the playable module files, there is nothing needed. The correct names need to appear in the source files, but the Resource files are completely separate.

Right. But if they need to correctly appear in the source files, then the compiler should have a way of checking that those files actually exist on the output folder.

Apart from that, I also noticed that the meshes have parameters on the source (e.g. scale). Therefore, the source is strictly related with the resources. In particular, adding a new item to the source most likely requires a new resource file, which implies that a specific version of the source (a commit) is bound to a specific version of resources. 

Therefore, it makes sense to have both source and resources together on git.
